### ISSUE TITLE
Timecodes for "ML Zoomcamp 3.3 - Setting Up The Validation Framework"

### ISSUE THREAD

**Issue by alexeygrigorev** [DataTalksClub/machine-learning-zoomcamp#206]
Youtube video: https://www.youtube.com/watch?v=_lwz34sOnSE

**Comment by dimzachar** [comment]
0:00:00 - Lesson 3: Setting up validation framework with Scikit-learn
0:01:19 - Specifying test size and fixing seeds.
0:02:43 - Splitting data into train, validation, test sets.
0:04:06 - Shuffling indices and getting y variables.

**Comment by amitfrancis** [comment]
Updated timecodes. Thanks!
